core/src/main/java/org/apache/struts2/dispatcher/Dispatcher.java

```diff
@@ -554,6 +566,7 @@ public void init() {
             init_CustomConfigurationProviders(); // [5]
             init_FilterInitParameters(); // [6]
             init_AliasStandardObjects(); // [7]
+            init_DeferredXmlConfigurations();
```

Comment:
Nice, do you plan document this functionality in the docs?

Reply:
I will create a struts-site PR adding a note about `struts-deferred.xml` to:
- https://struts.apache.org/plugins/plugins-architecture
- https://struts.apache.org/core-developers/struts-xml